bentoml/docker/images.py

```diff
@@ -0,0 +1,171 @@
+import logging
+import re
+from typing import Dict, List, Optional
+
+from bentoml import __version__ as BENTOML_VERSION
```

Comment:
@ssheng I was informed that we also have `bento_bundle_deployment_version` under `bentoml/configurations/containers.py`. However, I wasn't sure on how to implement DI for this. If you can provide some feedback that would be great.